snap/snapcraft.yaml

```diff
@@ -79,12 +79,45 @@ parts:
       # Replace key-value store path:
       sed -i 's/\/tmp/\/mnt/g' src/platform/Linux/CHIPPlatformConfig.h
 
+      # Install NodeJS and ZAP tool for arm builds
```

Comment:
This scriptlet is mainly taken from [this dockerfile.](https://github.com/project-chip/connectedhomeip/blob/master/integrations/docker/images/chip-cert-bins/Dockerfile)